.ci/docker/conan-tests

```diff
@@ -149,6 +149,9 @@ RUN mkdir -p /usr/share/bazel-$BAZEL_6/bin && \
     chmod +x bazel-${BAZEL_8}-linux-x86_64 && \
     mv bazel-${BAZEL_8}-linux-x86_64 /usr/share/bazel-$BAZEL_8/bin/bazel
 
+RUN wget https://github.com/premake/premake-core/releases/download/v5.0.0-beta4/premake-5.0.0-beta4-linux.tar.gz && \
+    tar -xvzf premake-5.0.0-beta4-linux.tar.gz && chmod +x premake5 && mkdir /usr/share/premake && \
+    mv premake5 /usr/share/premake
```

Comment:
Let's declare above something like:

```dockerfile
BAZEL_8=8.0.0 \
PREMAKE_5=5.0.0
```
So we can use it here as:
```dockerfile
RUN wget https://github.com/premake/premake-core/releases/download/v${PREMAKE_5}-beta4/premake-${PREMAKE_5}-beta4-linux.tar.gz && \
    tar -xvzf premake-${PREMAKE_5}-beta4-linux.tar.gz && chmod +x premake5 && mkdir /usr/share/premake && \
```

Reply:
I don't plan to add or support more than 1 premake version in Conan and CI, do you think it is still necessary?

Reply:
Not at all 👍